csm/conf/etc/rsyslog.d/1-csmlog.conf.tmpl

```diff
@@ -19,7 +19,7 @@ if $syslogtag == 'FILENAMETAG' then
     action(type="omfile"
            dirCreateMode="0700"
            FileCreateMode="0644"
-           File="/var/log/seagate/node/FILE")
+           File="/var/log/cortx/node/FILE")
```

Comment:
Use a tag here and Use value from conf store rather than hardcoded one.

Reply:
I don't see any references of this file in our code. Do we still need to keep this file or we can remove this?